sdk/core/Azure.Core/src/Serialization/ModelSerializerOptions.cs

```diff
@@ -35,5 +40,14 @@ public class ModelSerializerOptions
         /// NameHint for Xml Models
         /// </summary>
         public string? NameHint { get; set; }
+
+        private string ValidateFormat(string x)
+        {
+            if (x != "D" && x != "W")
```

Comment:
Can we make constants for these so we know what they mean?

Reply:
If `Format` can't be used with the other standard string formats, should we consider doing this as an Enum instead?